<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1171</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>811</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_fig">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>230</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Переместить модель</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_shift_fig">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_shift_fig">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_shift_fig">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>230</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_fig">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>260</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Повернуть модель</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="yz_angle_fig">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="scale_fig">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>290</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Смаштабировать модель</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_scale_fig">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_scale_fig">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_scale_fig">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>290</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="xy_angle_fig">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="zx_angle_fig">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>260</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="import_fig">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>160</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить модель</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>160</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="add_cam">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>10</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Загрузить камеру</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="xy_angle_cam">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="shift_cam">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>110</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Переместить камеру</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="zx_angle_cam">
    <property name="geometry">
     <rect>
      <x>1110</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="y_shift_cam">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="yz_angle_cam">
    <property name="geometry">
     <rect>
      <x>1060</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="x_shift_cam">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="z_shift_cam">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>80</y>
      <width>41</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="rotate_cam">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>110</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Повернуть камеру</string>
    </property>
   </widget>
   <widget class="QComboBox" name="camera_select">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>40</y>
      <width>141</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_cam">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>40</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить камеру</string>
    </property>
   </widget>
   <widget class="QPushButton" name="remove_fig">
    <property name="geometry">
     <rect>
      <x>1010</x>
      <y>190</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Удалить модель</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="file_name_camera">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>10</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="nums_figs">
    <property name="geometry">
     <rect>
      <x>850</x>
      <y>190</y>
      <width>141</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1171</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
